modules/dbparser/groupingby.c

```diff
@@ -383,6 +397,25 @@ grouping_by_process(LogParser *s, LogMessage **pmsg, const LogPathOptions *path_
   return TRUE;
 }
 
+static void
+_load_data_from_persist(GroupingBy *self, GlobalConfig *cfg)
```

Comment:
This does not just load data from persist, it creates state when the persist is empty, so it would be the best if the name could reflect this.

What do you think of `_load_correlation_state` (TimerWheel is actually coupled with the correlation state) or `_load_grouping_by_state` or `_load_state`?

Reply:
Sounds good to me. I pick `_load_correlation_state`.

Reply:
Thanks!